Info.lua

```diff
@@ -2,12 +2,13 @@ return {
     LrSdkVersion = 3.0,
     LrSdkMinimumVersion = 3.0, -- minimum SDK version required by this plug-in
     LrToolkitIdentifier = 'au.id.willthames.photodeck',
-    LrPluginName = 'PhotoDeck Publisher',
+    LrPluginName = 'PhotoDeck',
```

Comment:
I don't believe this is part of the plugin key in the Lightroom catalog. Don't remember seeing any effect after having changed this either.
But we can revert that commit if you don't like this change ;)
The rational behind this change was that the plugin can also be used for simple export operations (so the name "PhotoDeck Published" was a bit confusing in that case).